Refactor Code

julzedz opened the pull request:
In this project, we reviewed our code to make sure that:
- We followed SOLID Principles.
- YAGNI, KISS and DRY rules are respected.
- The role of main.rb is only to start code and ask the user for the initial choice of options.

Commit:
Refactor

Qoosim reviewed (commented):
#  *Status: Invalid Request* :red_circle: 

Hi @team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

![](https://user-images.githubusercontent.com/97100091/194103659-a0270409-def7-4972-b884-9f7c141eb90a.gif)

## Invalid Code Review Request

You have submitted a project, that is a pair-programming but has commit only from one partner. Kindly ensure that every member of the team has at least one commit in the PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

VelzckC0D3 committed:
require relative

Updated the code to follow best practices, now the files that are on the same root file instead of "require" have "require_relative"

AnselemOdims reviewed (approved):
### Hi  @team🙋‍♀️, 
Good job implementing all the project requirements for this milestone/project.

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
